### DISCUSSION TITLE
Be able to map `C-j`?

### DISCUSSION THREAD

**Discussion by malkomalko** [wezterm/wezterm#1072] [Q&A] [answered]
Is there anyway to have `C-j` not be a line feed?  I've been looking in the docs and have tried a few config changes unsuccessfully.  I'd like to be able to have that key to be able to be mapped in vim (as an example).

**Comment by wez** [comment, marked as answer]
CTRL-j is functionally identical to line feed in the standard terminal input encoding.

https://github.com/wez/wezterm/issues/63 discusses the incomplete implementation of a non-standard alternative input encoding that can distinguish between the two, that you can enable using `enable_csi_u_key_encoding = true`.

**Reply by malkomalko** [reply]
Thank you for the response. 🙇🏻
